### PR TITLE
disable incorrect tests

### PR DESCRIPTION
**Pull request by brettfo**
Skips some incorrect tests as detailed in #6523.

FYI @tmat @tannergooding @amcasey @davkean

### CODE REVIEW

File: build/ToolsetPackages/project.lock.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
           "lib/net45/Microsoft.CodeAnalysis.Test.Resources.Proprietary.dll": {}
         }
       },
-      "Microsoft.DiaSymReader.Native/1.2.0-rc": {},
```
Comment (amcasey) [inline build/ToolsetPackages/project.lock.json]: Did you intend to change this file?

**Comment by amcasey** [comment]
:+1:

File: build/ToolsetPackages/project.lock.json (hunk 1 above)
Comment (brettfo) [inline build/ToolsetPackages/project.lock.json]: Yes, that's expected to keep it up to date.

**Comment by tmat** [comment]
Could you just replace the version with {CompilerVersion}? Thanks

**Pull request merged**